.github/workflows/build-zarf-agent.yml

```diff
@@ -12,29 +12,72 @@ on:
         default: "master"
 
 jobs:
+<<<<<<< HEAD
```

Comment:
@jeff-mccoy looks like GitKraken has failed you again :(

Reply:
Ughhhh it totally did...